libindy/src/commands/did.rs

```diff
@@ -490,6 +502,79 @@ impl DidCommandExecutor {
         Ok(res)
     }
 
+    fn qualify_did(&self,
+                   wallet_handle: WalletHandle,
+                   did: &str,
+                   prefix: &str) -> IndyResult<String> {
+        info!("qualify_did >>> wallet_handle: {:?}, curr_did: {:?}, prefix: {:?}", wallet_handle, did, prefix);
+
+        self.crypto_service.validate_did(did)?;
+
+        let prefix_parts = prefix.split_terminator(":").collect::<Vec<&str>>().len();
```

Comment:
This looks pretty weird. I would expect passing `pref` to the method, not the `did:pref`.